backend/routers/index_router.py

```diff
@@ -23,7 +23,7 @@ async def read_main():
     return {"msg": "Hello World"}
 
 
-@index_router.get("/get_users", response_model=List[schemas.User])
-def get_users(db: Session = Depends(get_db)):
-    users = db.query(models.User).all()
+@index_router.get("/get_pixel_watch", response_model=List[schemas.PixelWatch])
```

Comment:
Imo this should be in different router

Reply:
Hmm, maybe I should just remove this route?

Reply:
Whatever idk what it has to to do with this pr